### ISSUE TITLE
vWii does not load Wii games from 5.5.5U Wii U's microSD

### ISSUE THREAD

**Issue by takoyaki99** [Fledge68/WiiFlow_Lite#291]
**Describe the bug**
I have my Wii U hacked with CBHC and Homebrew Channel installed into my vWii.
I am unable to load Wii games, and have tested this with a few different games that match Redump hashes.
I've installed the relevant cIOSes, including the vWii IOS80.

The screen, after clicking "Play", boots to black and cannot be powered off by a 4-second forced power-off, only by unplugging the console. I've tried this with IOS 249, 250, and 251, with force IOS switched from Auto to On. Also tried with IOS 251 and Off to the same result.

**To Reproduce**
1. Go to Homebrew Channel on CBHC Wii U
2. Click on the 5.5.0 release of Wiiflow Lite
3. Play a WBFS Redump game
4. Black screen after hitting "Play" with no white loading bar

**What version of WiiFlow Lite**
v5.5.0

**On Wii or Wii U vWii**
vWii

**Additional context**

Used WiiFlow on my Wii normally without problems, got a Wii U for Christmas and hacked it yesterday (same game files used on Wii). 
I'm loading games and homebrew from a 200GB Sandisc microSD in an adapter

**Comment by Fledge68** [comment]
try v5.4.9 - [https://github.com/Fledge68/WiiFlow_Lite/releases/download/v5.4.9/wiiflow_v5.4.9.zip](url)

let me know if it works with your SD card.

**Comment by takoyaki99** [comment]
v5.4.9 seems to work well, I've tested it out with two games so far and it runs normally.
I have Force Load cIOS to Off and Force cIOS Revision to 251.
Thank you very much for the help!

**Comment by Fledge68** [comment]
thats great but your missing all the new stuff in v5.5.0. so hopefully i can figure out why v5.5.0 is having trouble launching games from SD

**Comment by takoyaki99** [comment]
> thats great but your missing all the new stuff in v5.5.0. so hopefully i can figure out why v5.5.0 is having trouble launching games from SD

If you need, I can help test out fixes/betas and that sort of thing. I can also contribute a syscheck for my vWii!

**Comment by takoyaki99** [comment]
Did some testing by myself, and I have something to report!
So they worked fine, and I rebooted to the Wii U menu and did some other stuff,
Later on I tried to boot some game and the black screen issue happened again (on v5.4.9).
And I was like aaah, maybe I installed the cIOSes incorrectly? So I decided to use vWii Decaffeinator on Advanced Mode to nuke/wipe the vWii NAND and reinstall everything, start new.
I did it, and rebooted the console, and got a scary error message! I thought, wait, did I somehow brick my CBHC setup? oh no. ...Then I realized that my error message issue was caused by a Wii disc inside the system, so I removed the disc and updated fine.
Anyways, I didn't say this before, but I used Compat Installer to install the Homebrew Channel to my vWii.
I used SysCheck to see what was up, just incase if something weird was going on (before I wiped the NAND). I don't have the file anymore, it's uploaded to the [website ](https://syscheck.rc24.xyz/)but I have no idea how to access it.
I remember that the file said that I did **not** have the Homebrew Channel installed, despite having it installed with Compat Installer.

So I ended up deciding to install it with the conventional Wuphax method. Wuphax actually uses the Hackmii installer, unlike Compat Installer which somehow installs the OHBC version of the Homebrew Channel (without the Hackmii installer, the channel just shows up), instead of Hackmii's LULZ version.

Games booted completely fine once I got the Homebrew Channel installed (and my cIOSes installed as well).

So, I thought, maybe I should try using 5.5.0 this time and see if it works.

And it works completely fine with 5.5.0, with `Force Load cIOS` to `Off `and `Force cIOS revision` to `251`.

I've tested it two separate times, with two separate games. Both load fine.

TL;DR: So, I don't know if this is a reproducible error or not, but [Compat Installer's](https://github.com/TheLordScruffy/vwii-compat-installer) Homebrew Channel may mess with the way homebrew software works/loads.

**Comment by Fledge68** [comment]
interesting ... i never heard of compat installer. 

So force load cios to ON doesn't work?  and what cios is installed in slot 251?

to view your syscheck you and I need the syscheck ID. do you have it?

glad you got it working. if the problem comes back let me know.

**Comment by takoyaki99** [comment]
> interesting ... i never heard of compat installer.
It's used in the [wiiu.hacks.guide](url) vWii hacking section now, it's very new.
> So force load cios to ON doesn't work? and what cios is installed in slot 251?
251 cIOS is `d2x-v10beta52-vWii`, and `Force Load cIOS` to `ON` actually works for me, I just tested it out.
> to view your syscheck you and I need the syscheck ID. do you have it?
Sadly no, I've tried multiple times with different SysCheck and the ID is not given to me after uploading, sadly.
> glad you got it working. if the problem comes back let me know.
Will do. If more issues come from people who installed with Compat Installer then it might be good to ask them for a SysCheck.
This may be an issue that was only for me for whatever reason, but using vWii Decaffeinator and installing with Wuphax worked in the end.

If this issue happens with other people as well just let me know and I'll do whatever I can to help you out!
Thank you very much!

[My sysCheck after installing with Wuphax](https://github.com/Fledge68/WiiFlow_Lite/files/7786447/sysCheck.csv)

**Comment by mastercreeper66** [comment]
Hi, I'm having the same problem (black screen on 5.5.0 and on 5.5.1) but on 5.4.9 It gives [Exception (DSI) occurred](https://user-images.githubusercontent.com/31628599/149633181-43ecd070-a18d-4b07-b684-c69f07f16079.jpg)

I used wuphax to mod the vWii

Here's my [sysCheck.csv](https://github.com/Fledge68/WiiFlow_Lite/files/7875525/sysCheck.csv)

**Comment by peytonzhu** [comment]
> Hi, I'm having the same problem (black screen on 5.5.0 and on 5.5.1) but on 5.4.9 It gives [Exception (DSI) occurred](https://user-images.githubusercontent.com/31628599/149633181-43ecd070-a18d-4b07-b684-c69f07f16079.jpg)
> 
> I used wuphax to mod the vWii
> 
> Here's my [sysCheck.csv](https://github.com/Fledge68/WiiFlow_Lite/files/7875525/sysCheck.csv)

My wiiu 5.5.5J has the same problem
